src/ert/ensemble_evaluator/_builder/_legacy.py

```diff
@@ -45,7 +45,7 @@ def __init__(
         if not queue_config:
             raise ValueError(f"{self} needs queue_config")
 
-        if FeatureToggling.is_enabled("scheduler"):
+        if FeatureToggling.value("scheduler") == "new":
```

Comment:
`"scheduler" == "new"` what's the idea behind these? I thought the `scheduler` is already `new` queue.

Reply:
changed to True/False/None